cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -106,6 +106,7 @@ public final boolean install(boolean silent, EnvironmentContext environmentConte
       } else {
         step.success("Successfully installed {} in version {} replacing previous version {}", this.tool, resolvedVersion, installedVersion);
       }
+      postInstall();
```

Comment:
`postInstall` is already called above in line 103. Please remove this.
```suggestion
```
See also 
https://github.com/devonfw/IDEasy/blob/73c1efcb20e441201e80356fb261b2ded122548f/cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java#L242-L247